operator/pkg/gateway-api/gamma_status.go

```diff
@@ -0,0 +1,88 @@
+package gateway_api
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	CiliumGammaPrefix = "gamma.cilium.io"
+)
+
+const (
+	CiliumGammaConditionAccepted = CiliumGammaPrefix + "/GammaRoutesAttached"
+
+	CiliumGammaReasonAccepted = "Accepted"
+)
+
+const (
+	CiliumGammaConditionProgrammed = CiliumGammaPrefix + "/GammaRoutesProgrammed"
+
+	CiliumGammaReasonProgrammed = "Programmed"
+)
+
+const (
+	CiliumGammaConditionResolvedRefs = CiliumGammaPrefix + "/GammaRoutesResolvedRefs"
+
+	CiliumGammaReasonResolvedRefs = "UnresolvedReference"
+)
```

Comment:
Those are actually unused, can we get rid of them for now?

Reply:
Sure, but they'll definitely come back soon.